eng/pipelines/stress-test-release.yml

```diff
@@ -0,0 +1,72 @@
+pr: none
+
+trigger: none
+
+parameters:
+  - name: Subscription
+    type: string
+    default: 2cd617ea-1866-46b1-90e3-fffb087ebf9b
+  - name: Environment
+    type: string
+    default: prod
+  - name: ClusterName
+    type: string
+    default: stress-prod
+  - name: ClusterGroup
+    type: string
+    default: rg-stress-test-cluster-prod
+  - name: RegistryName
+    type: string
+    default: stressprodregistry
+
+jobs:
+- job: ReleaseStressTests
+  strategy:
+    matrix:
+      examples:
+        Repository: Azure/azure-sdk-tools
+        Filters: '@{ "example" = "true" }'
+      javascript:
+        Repository: Azure/azure-sdk-for-js
+        Filters: '@{}'
+      java:
+        Repository: Azure/azure-sdk-for-java
+        Filters: '@{}'
+      net:
+        Repository: Azure/azure-sdk-for-net
+        Filters: '@{}'
+      python:
+        Repository: Azure/azure-sdk-for-python
+        Filters: '@{}'
+  pool:
+    name: 'azsdk-pool-mms-ubuntu-2004-general'
+    vmImage: 'MMSUbuntu20.04'
+  steps:
+    - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+      parameters:
+        Repositories:
+          - Name: Azure/azure-sdk-tools
+            WorkingDirectory: $(System.DefaultWorkingDirectory)/Azure/azure-sdk-tools
+          - Name: $(Repository)
+            WorkingDirectory: $(System.DefaultWorkingDirectory)/$(Repository)
+        Paths:
+          - '!sdk/**/test-recordings'
+          - '!sdk/**/session-records'
+          - '!sdk/**/SessionRecords'
+
+    - task: AzureCLI@2
```

Comment:
What value are we gaining by using the AzureCLI task here? Is the service connection information enough for us to do all the deployments or don't we need to stull pass in most of the secrets? Also what about other clouds do they need other configurations or is that not a scenario at this point.

Reply:
I hadn't gotten to that point yet but my thinking was to add an access policy to the buildout bicep configs to allowlist one of the devops service connections (that way there are no manual steps on buildout). I may still create a new identity just for this.

I think it's unlikely that we'll target other clouds, as our aim here is to stress the client code, rather than to test its operability across clouds. Though if we do have a need for that, I don't think we'll ever need to host the stress cluster itself in another cloud, so any relevant credentials are not configured in this context (they are auto-synced into the cluster via keyvault).